### PR TITLE
docs: update Upstream`s Request Body Parameters add lost Type colum a…

### PR DESCRIPTION
**Pull request by dyrnq**
docs: update Upstream`s Request Body Parameters add lost Type colum and rename first and second colum as same as others.

### Description

Fixes # (issue)

### Checklist

- [ ] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [x] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

**Comment by moonming** [comment]
@kayx23 please take a look

**Comment by kayx23** [comment]
@dyrnq thanks for the contribution. Please resolve the merge conflicts.

**Review by kayx23 — CHANGES_REQUESTED** [review]
@dyrnq thanks for the contribution. Please resolve the merge conflicts.

**Comment by dyrnq** [comment]
resolve the merge conflicts

File: docs/en/latest/admin-api.md [hunk 1]
```diff
@@ -887,30 +887,30 @@ For notes on ID syntax please refer to: [ID Syntax](#quick-note-on-id-syntax)
...
+| tls.client_key              | False, can't be used with `tls.client_cert_id`                   | https certificate private key | Sets the client private key while connecting to a TLS Upstream.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |                                                                                                                                            |
```
Comment (kayx23) [inline docs/en/latest/admin-api.md:909]:
```suggestion
| tls.client_key              | False, can't be used with `tls.client_cert_id`                   | HTTPS certificate private key | Sets the client private key while connecting to a TLS Upstream.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |                                                                                                                                            |
```

File: docs/en/latest/admin-api.md [hunk 2]
```diff
@@ -887,30 +887,30 @@ For notes on ID syntax please refer to: [ID Syntax](#quick-note-on-id-syntax)
...
+| service_name                | True, can't be used with `nodes`                                 | string                        | Service name used for [service discovery](discovery.md).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     | `a-bootiful-client`                                                                                                                        |
```
Comment (kayx23) [inline docs/en/latest/admin-api.md:894]:
```suggestion
| service_name                | True, can't be used with `nodes`                                 | String                        | Service name used for [service discovery](discovery.md).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     | `a-bootiful-client`                                                                                                                        |
```

File: docs/en/latest/admin-api.md [hunk 3]
```diff
@@ -887,30 +887,30 @@ For notes on ID syntax please refer to: [ID Syntax](#quick-note-on-id-syntax)
...
+| discovery_type              | True, if `service_name` is used                                  | string                        | The type of service [discovery](discovery.md).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               | `eureka`                                                                                                                                   |
```
Comment (kayx23) [inline docs/en/latest/admin-api.md:895]:
```suggestion
| discovery_type              | True, if `service_name` is used                                  | String                        | The type of service [discovery](discovery.md).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               | `eureka`                                                                                                                                   |
```

File: docs/en/latest/admin-api.md [hunk 4]
```diff
@@ -887,30 +887,30 @@ For notes on ID syntax please refer to: [ID Syntax](#quick-note-on-id-syntax)
...
+| checks                      | False                                                            | health_checker                | Configures the parameters for the [health check](./tutorials/health-check.md).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |                                                                                                                                            |
```
Comment (kayx23) [inline docs/en/latest/admin-api.md:898]:
```suggestion
| checks                      | False                                                            | Health Checker                | Configures the parameters for the [health check](./tutorials/health-check.md).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |                                                                                                                                            |
```

**Review by kayx23 — APPROVED** [review]

**Review by monkeyDluffy6017 — APPROVED** [review]

**Pull request merged**